scripts/format_code.py

```diff
@@ -175,14 +173,31 @@ def validate_arguments():
       print('ERROR:  -git_diff not defined, and there are no file or')
       print('directory search targets.')
       print('Nothing to do. Exiting.')
-      print()
       return False
   return True
 
+def is_file_objc_header(filename):
+  """Checks the contents of the file to determine if it contains language 
+  constructs that appear in obj-c but not in C/C++.
+  
+  Args:
+    filename (string): the name of the file to check
+  
+  Returns:
+    bool: True if the header file contains known obj-c language definitions.
+    Returns False otherwise.
+  """
+  if '.h' not in filename:
+    return False
+  objective_c_tags = [ '@end', '@class', '#import']
+  for line in open(filename):
```

Comment:
Better to use context manager as that will take care of closing the file descriptor automatically.
```
with open(filename) as some_file:
   lines = some_file.readlines()
   for line in lines:
       ...
```

Reply:
Done.